tools/tcpretrans.py

```diff
@@ -152,6 +219,23 @@ class Data_ipv6(ct.Structure):
 type[1] = 'R'
 type[2] = 'L'
 
+# cnt data
+class Flow_key_ipv4(ct.Structure):
+    _fields_ = [
+        ("saddr", ct.c_ulonglong),
+        ("daddr", ct.c_ulonglong),
+        ("lport", ct.c_ulonglong),
+        ("dport", ct.c_ulonglong)
+    ]
+
+class Flow_key_ipv6(ct.Structure):
+    _fields_ = [
+        ("saddr", (ct.c_ulonglong * 2)),
+        ("daddr", (ct.c_ulonglong * 2)),
+        ("lport", ct.c_ulonglong),
+        ("dport", ct.c_ulonglong)
+    ]
+
```

Comment:
Do you need these two types? These two types should be inferred when you call get_table() and further to reference keys through table pointer?

Reply:
oliviertilmans already pointed that out. No, gone!